homeassistant/components/binary_sensor/strings.json

```diff
@@ -243,8 +243,8 @@
     "power": {
       "name": "Power",
       "state": {
-        "off": "[%key:component::binary_sensor::entity_component::gas::state::off%]",
-        "on": "[%key:component::binary_sensor::entity_component::gas::state::on%]"
+        "off": "[%key:common::state::off%]",
+        "on": "[%key:common::state::on%]"
```

Comment:
You can remove both as `off` and `on` are default values for a binary sensor, so we don't need to overwrite the default state with the same values

Reply:
I think it is fine to be explicit in this case 👍